Off-Top-App/off-top-python
# OT-89 | Create Documentation to set up Python and Java Instance

**What I Did:**
1. I created documentation in Java and Python. 
2. Included a scaffold text file that when running a command you are able to install all required python modules

**How To Test:**
1. If you do not have our Python instance set up on your machine you will need to follow this documentation (https://docs.google.com/document/d/1iF12gqeqYPhHOIWnWl610keb653pw3FoGgPD5LCXf8U/edit) to make sure that my documentation is sufficient enough. 
2. If you're stuck on something for too long from my documentation please comment where and message me so I can further the information as well as aiding you.
